trackintel/io/__init__.py

```diff
@@ -29,3 +29,4 @@
 from .from_geopandas import read_trips_gpd
 
 from .dataset_reader import read_geolife
+from .dataset_reader import geolife_add_modes_to_triplegs
```

Comment:
I don't see this `geolife_add_modes_to_triplegs` being used anywhere in this list of changes. Wondering if it serves any purpose?

Reply:
if we do not include a function in `init.py`, the function will not be able to call outside this module. e.g., using `from trackintel.io.dataset_reader import geolife_add_modes_to_triplegs` will raise an error.
So if we want a function to be called by the user ("outside" function), we need to add it to the corresponding `init.py` file